package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mondorepo",
-  "version": "0.1.701",
+  "version": "0.1.702",
```

Comment:
The change in this PR will drastically change the way `mondo install` behaves (it used to be including forks by default). Shouldn't we bump at least a minor version?